addons/l10n_in/views/account_journal_views.xml

```diff
@@ -11,6 +11,9 @@
             <xpath expr="//field[@name='loss_account_id']" position="after">
                 <field name="l10n_in_import_export" attrs="{'invisible': [('type', 'not in', ['sale', 'purchase'])]}"/>
             </xpath>
+            <field name="company_id" position="after">
+                <field name="l10n_in_gstin_partner_id" context="{'show_vat':True}" options='{"no_create": True,"always_reload": True}'/>
```

Comment:
always_reload needed?

Reply:
because when using show_vat in context then always_reload needed.